Source/Widgets/Viewer/Viewer.js

```diff
@@ -703,7 +703,7 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         }
 
         cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndSelectObject, ScreenSpaceEventType.LEFT_CLICK);
-        cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndTrackObject, ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
+        cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndTrackObject, ScreenSpaceEventType.MIDDLE_CLICK);
```

Comment:
I figure this will be the biggest breaking change. If this is commonly used it may be ok to keep LEFT_DOUBLE_CLICK.